docs/docs/howto/installation/configure-klaw-wizard.rst

```diff
@@ -0,0 +1,87 @@
+Configure Klaw with Wizard
+==========================
+
+When Klaw is installed for the first time, it prompts the Administrators with a Wizard to configure and make it available for the users.
+
+.. note::  Wizard is accessible at : `https://<host>:<port>/helpwizard`
+
+.. image:: /../../../_static/images/KlawWizard.png
+
+1. Configure Kafka clusters and update Klaw Cluster Api properties with relevant config
+
+You can add any number of clusters. Only users with SUPERADMIN roles(or check allowed permission) can add.
+
+After adding the cluster, if you have selected SSL or any other protocol, make sure Cluster Api properties are configured.
+
+Example ::
+
+    dev1.kafkassl.protocol=TLS
+    dev1.kafkassl.key.pwd=trustme
+    dev1.kafkassl.keystore.location=/srv/certs/client.keystore.p12
+    dev1.kafkassl.keystore.pwd=trustme
+    dev1.kafkassl.keystore.type=PKCS12
+    dev1.kafkassl.truststore.location=/srv/certs/client.truststore.jks
+    dev1.kafkassl.truststore.pwd=trustme
+    dev1.kafkassl.truststore.type=JKS
+
+2. Configure Environments
+
+Environments are wrappers over your kafka clusters. You can have multiple environments connected to one Kafka cluster with different settings
+
+You can add any number of environments. Only users with SUPERADMIN roles (or check allowed permission) can add by default.
+
+You can configure min max partitions, replication factors, prefix or suffix for topic names.
```

Comment:
"min max partitions" doesn't sound like a sentence. Maybe "minimum or maximum number of partitions"?

Reply:
Updated.